foreman_scc_manager.gemspec

```diff
@@ -24,5 +24,5 @@ Gem::Specification.new do |s|
   s.add_development_dependency 'webmock'
 
   s.add_dependency 'foreman-tasks', '>= 0.10'
-  s.add_dependency 'rails', '~> 5.1'
+  s.add_dependency 'rails', '>= 5.1'
```

Comment:
Would it be better to specify the katello version? https://github.com/ATIX-AG/foreman_scc_manager/blob/15ec2971ccbbd566581aeb2c40a0c14991539f8e/lib/foreman_scc_manager/engine.rb#L26 already states this is needed and katello implies a minimum rails version as well.

Reply:
I like the idea, because the plugin will not work without katello.
However, in outher foreman-plugins like katello I also do not see a dependency to foreman in the gemspec.

But, feel free to open a PR to change the gem-dependency to katello so we can have a discussion on it.

Reply:
We never depend on Foreman itself because it's not a gem, so you can't. However, Katello is a gem so you can (and should) depend on it.